### PR TITLE
Adding zoxide

### PR DESCRIPTION
**Pull request by daizeng1984**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by daizeng1984** [comment]
I'm willing to maintain

**Comment by daizeng1984** [comment]
@conda-forge/staged-recipes please review

File: recipes/zoxide/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+test:
+  commands:
+    - zoxide --help
+
+about:
+  home: https://github.com/ajeetdsouza/zoxide
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```
Comment (chrisburr) [inline recipes/zoxide/meta.yaml:29]: The licenses of the dependencies need to be included for rust packages. See https://github.com/conda-forge/conda-forge.github.io/issues/1052 for a starting point

Comment (daizeng1984) [reply recipes/zoxide/meta.yaml:29]: trying to follow [this](https://github.com/conda-forge/staged-recipes/pull/13208#issuecomment-726767185)

**Comment by daizeng1984** [comment]
```
missing libray licenses:                                                                            
('clap', '3.0.0-beta.2', 'Apache-2.0 OR MIT', 'https://github.com/clap-rs/clap')                    
('clap_derive', '3.0.0-beta.2', 'Apache-2.0 OR MIT', 'https://github.com/clap-rs/clap/tree/master/clap_derive')
('remove_dir_all', '0.5.3', 'Apache-2.0 OR MIT', 'https://github.com/XAMPPRocky/remove_dir_all.git') 
('winapi-i686-pc-windows-gnu', '0.4.0', 'Apache-2.0 OR MIT', 'https://github.com/retep998/winapi-rs')
('winapi-x86_64-pc-windows-gnu', '0.4.0', 'Apache-2.0 OR MIT', 'https://github.com/retep998/winapi-rs')
52 having 'LICENSE-MIT', 11 having 'LICENSE*' or custom filename, 5 licenses are missing!!!; in total 68
```

**Comment by daizeng1984** [comment]
Manually downloads:
```
Only in library_licenses/: clap-3.0.0-beta.2-license
Only in library_licenses/: clap_derive-3.0.0-beta.2-license
Only in library_licenses/: remove_dir_all-0.5.3-license
Only in library_licenses/: winapi-i686-pc-windows-gnu-0.4.0-license
Only in library_licenses/: winapi-x86_64-pc-windows-gnu-0.4.0-license

```

File: recipes/zoxide/meta.yaml (hunk 1 above)
Comment (daizeng1984) [reply recipes/zoxide/meta.yaml:29]: @chrisburr done with the license files. Please take a look, thanks!

**Pull request merged**